package.json

```diff
@@ -33,7 +33,7 @@
     "utils.js"
   ],
   "scripts": {
-    "lint": "standard",
+    "lint": "standard --fix",
```

Comment:
I'm sure this is used for checking for the lint issues, since it's referenced from `test` script. As far as I understand it would break dead CI integration.

Reply:
Yeah, `lint` shouldn't run `--fix`